wordle-web/components/NameDialog.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <v-dialog v-model="showModel" max-width="350" persistent>
+    <v-card>
+      <v-sheet color="secondary">
+        <v-card-title>Enter your name:</v-card-title>
+      </v-sheet>
+      <v-form>
+        <v-responsive hide-details="auto" width="350">
+          <v-text-field
+            append-inner-icon="mdi-arrow-right-bold-circle"
+            label="Name"
+            v-model="nameModel"
+            @keydown.enter
+            @click:append-inner="$emit('entered')"
+            @keydown.enter.prevent></v-text-field>
+        </v-responsive>
+      </v-form>

```

Comment:
```suggestion
          <v-text-field
            append-inner-icon="mdi-arrow-right-bold-circle"
            label="Name"
            v-model="nameModel"
            @keydown.enter
            @click:append-inner="$emit('entered')"
            @keydown.enter.prevent
          />
```

The `v-responsive` and `v-form` tags seem to serve no purpose. You can also use self closing tag on text field

Reply:
Thank you!